src/main/java/com/rakovets/course/java/core/practice/looping_statements/Task03.java

```diff
@@ -36,6 +36,11 @@ public static void main(String[] args) {
     static int calculateHealthPointsByTime(int healthPoints, double regenerationPercentFromCurrentHealth, int hoursAfterRespawn) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+        double healthPointsByTime = healthPoints;
+            for (int i = 0; i < hoursAfterRespawn; i++) {
+                double restoredHealth = healthPointsByTime * regenerationPercentFromCurrentHealth / 100;
+                healthPointsByTime += restoredHealth;
+            }
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"